src/leaf.ts

```diff
@@ -0,0 +1,120 @@
+const completionSpec: Fig.Spec = {
+  name: "leaf",
+  description: "",
+  options: [
+    {
+      name: ["-h", "--help"],
+      description:
+        "Display help for the given command. When no command is given display help for the list command",
+    },
+    {
+      name: ["-q", "--quiet"],
+      description: "Do not output any message",
+    },
+    {
+      name: ["-V", "--version"],
+      description: "Display this application version",
+    },
+    {
+      name: ["--ansi", "--no-ansi"],
+      description: "Force (or disable --no-ansi) ANSI output",
+    },
```

Comment:
These are two different options, one opposite of the other and must be listed separately:
```suggestion
    {
      name: "--ansi",
      description: "Force ANSI output",
      exclusiveOn: ["--no-ansi"],
    },
    {
      name: "--no-ansi",
      description: "Disable ANSI output",
      exclusiveOn: ["--ansi"],
    },
```